src/components/part/QueuedTranslationsTable.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  Box,
+  CircularProgress,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from "@material-ui/core";
+import React from "react";
+import useSWR from "swr";
+
+import { QueuedJob, toQueuedJobPage } from "../../lib/queued-jobs";
+
+interface QueuedTranslationsTableProps {
+  readonly status: string;
+  readonly refreshInterval?: number;
+  readonly title: string;
+  readonly filter?: (arg0: QueuedJob) => boolean;
+}
+
+async function fetcher(req: RequestInfo) {
```

Comment:
Can we move this to `api.ts` and remove it from the 4 or 5 places it is currently?